test/studies/adventOfCode/2022/day11/bradc/day11-par.chpl

```diff
@@ -0,0 +1,115 @@
+use IO, List, Set, Barriers;
```

Comment:
Should the day11 programs be in this PR too?

Reply:
No, that was an operator error that I think was caused by branching from my day11 branch, and should resolve itself once I start merging these in order...